bin/main.rb

```diff
@@ -1,16 +1,22 @@
 #!/usr/bin/env ruby
 require 'io/console'
+require 'colorize'
+require '../lib/board'
```

Comment:
Please use require_relative in this case, you can read more about why you should do that  [Here](https://medium.com/@ellishim/understanding-require-vs-require-relative-vs-require-all-80e3b26d89e6)